src/App.js

```diff
@@ -1,5 +1,65 @@
+import { Console } from '@woowacourse/mission-utils';
+import Racer from './Racer.js';
+import Racing from './Racing.js';
+import {
+  FIRST_USER_INPUT_PLACEHOLDER,
+  SECOND_USER_INPUT_PLACEHOLDER,
+  NAME_SEPARATOR,
+  OUTPUT_PREFIX,
+} from './constants/index.js';
+import Winners from './Winners.js';
+import Validator from './Validator.js';
+
 class App {
-  async run() {}
+  #totalRound = 0;
+
+  #racers = [];
+
+  async run() {
+    await this.#setRacers();
+    await this.#setTotalRound();
+
+    this.#startRacing();
+
+    const winners = this.#getWinners();
+
+    Console.print(`${OUTPUT_PREFIX} ${winners}`);
+  }
+
+  async #setRacers() {
+    const firstUserInput = await Console.readLineAsync(
+      FIRST_USER_INPUT_PLACEHOLDER,
+    );
+
+    Validator.validateRacers(firstUserInput);
+    const racers = firstUserInput.split(NAME_SEPARATOR).map((racer) => {
+      Validator.validateMaxRacerName(racer);
+      return new Racer(racer);
+    });
+
+    this.#racers = racers;
+  }
+
+  async #setTotalRound() {
+    const secondUserInput = await Console.readLineAsync(
+      SECOND_USER_INPUT_PLACEHOLDER,
+    );
+
+    Validator.validateTotalRound(secondUserInput);
+    this.#totalRound = secondUserInput;
```

Comment:
```suggestion
    this.#totalRound = Number(secondUserInput);
```
이렇게 형변환을 하지 않으신 이유가 있으신가요? 이 `this.#totalRound`가 결국 `Racing` 클래스에서 사용될 때 `number` 형으로 쓰이게 될 것 같아서요..!